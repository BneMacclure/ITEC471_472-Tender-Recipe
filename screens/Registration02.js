import React, { Component, useState } from "react";
import {
  StyleSheet,
  View,
  Image,
  ImageBackground,
  Text,
  TouchableOpacity,
  Dimensions
} from "react-native";

import styles from '../styles/RegisterScreenStyles.js';
const SCREEN_HEIGHT = Dimensions.get('window').height
const SCREEN_WIDTH = Dimensions.get('window').width

function Registration02({navigation, route}) {
  const [skill, setSkill] = useState("Beginner");
  const {name, email, phone, password} = route.params;
  const [btn1Act, setbtn1Act] = useState(false);
  const [btn2Act, setbtn2Act] = useState(false);
  const [btn3Act, setbtn3Act] = useState(false);

  const registerFunc2 = () => {
    navigation.navigate('Registration03', { name: name, email: email, phone: phone, password: password, skillLevel: skill })
  }

  const setBtns = (btnNum) => {
    setbtn1Act(btnNum == 1? true: false); 
    setbtn2Act(btnNum == 2? true: false);
    setbtn3Act(btnNum == 3? true: false);
  }

  return (
    <View style={styles.container}>
      <ImageBackground
        source={require("../assets/images/Gradient.png")}
        resizeMode="cover"
        style={styles.backImg}
        imageStyle={styles.backImg_imageStyle}
      >
        <Text style={styles.reg2Header}>What is your skill level?</Text>
        {/* Spacer */}
        <View style={{marginTop:100}}></View>
        <TouchableOpacity 
          onPress={() => {
            setBtns(1)
            setSkill("Beginner")
          }}
          style={btn1Act? styles.vertInpBtnActive : styles.vertInpBtnInactive}>
          <Text style={btn1Act? styles.inpBtnTxtActive : styles.inpBtnTxtInactive}>Beginner</Text>
        </TouchableOpacity>
        <TouchableOpacity 
          onPress={() => {
            setBtns(2)
            setSkill("Intermediate")
          }}
          style={btn2Act? styles.vertInpBtnActive : styles.vertInpBtnInactive}>
          <Text style={btn2Act? styles.inpBtnTxtActive : styles.inpBtnTxtInactive}>Intermediate</Text>
        </TouchableOpacity>
        <TouchableOpacity 
          onPress={() => {
            setBtns(3)
            setSkill("Advanced")
          }}
          style={btn3Act? styles.vertInpBtnActive : styles.vertInpBtnInactive}>
          <Text style={btn3Act? styles.inpBtnTxtActive : styles.inpBtnTxtInactive}>Advanced</Text>
        </TouchableOpacity>
        {/* Spacer */}
        <View style={{marginTop:200}}></View>
        <TouchableOpacity
            onPress={() => registerFunc2()}
            style={styles.nextBtn}>
          <Text style={styles.nextBtnTxt}>Next</Text>
        </TouchableOpacity>
      </ImageBackground>
    </View>
  );
  // return (
  //   <View style={styles.container}>
  //     <ImageBackground
  //       source={require("../assets/images/Gradient.png")}
  //       resizeMode="cover"
  //       style={styles.backImg}
  //       imageStyle={styles.backImg_imageStyle}
  //     >
  //       <Text style={styles.loremIpsum1}>What is your skill level?</Text>
  //       <TouchableOpacity
  //       style={styles.button}
  //       onPress={() => setSkill("Beginner")}>
  //         <Text style={styles.beginner}>Beginner</Text>
  //       </TouchableOpacity>
  //       <TouchableOpacity
  //       style={styles.button2}
  //       onPress={() => setSkill("Intermediate")}>
  //         <Text style={styles.intermediate}>Intermediate</Text>
  //       </TouchableOpacity>
  //       <TouchableOpacity
  //       style={styles.button3}
  //       onPress={() => setSkill("Advanced")}>
  //         <Text style={styles.advanced}>Advanced</Text>
  //       </TouchableOpacity>
  //       <TouchableOpacity
  //           onPress={() => registerFunc2()}
  //           style={styles.button4}>
  //         <Text style={styles.next1}>Next</Text>
  //       </TouchableOpacity>
  //     </ImageBackground>
  //   </View>
  // );
}

export default Registration02;