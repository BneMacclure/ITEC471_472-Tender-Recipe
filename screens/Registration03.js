import React, { Component, useState } from "react";
import { 
  StyleSheet, 
  View, 
  Image, 
  ImageBackground, 
  Text, 
  Dimensions, 
  TouchableOpacity,
 } from "react-native";

import styles from '../styles/RegisterScreenStyles.js';
const SCREEN_HEIGHT = Dimensions.get('window').height
const SCREEN_WIDTH = Dimensions.get('window').width

function Registration03({navigation, route}) {
  const [measurement, setMeasurement] = useState("Imperial");
  const {name, email, phone, skillLevel, password} = route.params;
  const [btn1Act, setbtn1Act] = useState(false);
  const [btn2Act, setbtn2Act] = useState(false);

  const registerFunc3 = () => {
    navigation.navigate('Registration04', { name: name, email: email, phone: phone, password: password, skillLevel: skillLevel, prefMeasurement: measurement })
  }

  const setBtns = (btnNum) => {
    setbtn1Act(btnNum == 1? true: false); 
    setbtn2Act(btnNum == 2? true: false);
  }

  return (
    <View style={styles.container}>
      <ImageBackground
        source={require("../assets/images/Gradient.png")}
        resizeMode="cover"
        style={styles.backImg}
        imageStyle={styles.backImg_imageStyle}
      > 
        <Text style={styles.reg2Header}>
          What is your preferred {"\n"} measurement units?
        </Text>
        <View style={styles.rectRow}>
          <TouchableOpacity 
            onPress={() =>{
              setBtns(1)
              setMeasurement("Metric")
            }}
            style={btn1Act? styles.horzInpBtnActive : styles.horzInpBtnInactive}>
            <Text style={btn1Act? styles.inpBtnTxtActive : styles.inpBtnTxtInactive}>Metric</Text>
          </TouchableOpacity>
          <TouchableOpacity 
            onPress={() => {
              setBtns(2)
              setMeasurement("Imperial")
            }}
            style={btn2Act? styles.horzInpBtnActive : styles.horzInpBtnInactive}>
            <Text style={btn2Act? styles.inpBtnTxtActive : styles.inpBtnTxtInactive}>Imperial</Text>
          </TouchableOpacity>
        </View>
        {/* Spacer */}
        <View style={{marginTop:330}}></View>
        <TouchableOpacity 
          onPress={() => registerFunc3()}
          style={styles.nextBtn}>
          <Text style={styles.nextBtnTxt}>Next</Text>
        </TouchableOpacity>
      </ImageBackground>
    </View>
  );

  // return (
  //   <View style={styles.container}>
  //     <ImageBackground
  //       source={require("../assets/images/Gradient.png")}
  //       resizeMode="cover"
  //       style={styles.image}
  //       imageStyle={styles.image_imageStyle}
  //     > 
  //       <Text style={styles.loremIpsum}>
  //         What is your preferred {"\n"} measurement units?
  //       </Text>
  //       <View style={styles.rectRow}>
  //         <TouchableOpacity
  //         style={styles.rect}
  //         onPress={() => setMeasurement("Metric")}>
  //           <Text style={styles.metric}>Metric</Text>
  //         </TouchableOpacity>
  //         <TouchableOpacity
  //         style={styles.rect1}
  //         onPress={() => setMeasurement("Imperial")}>
  //           <Text style={styles.imperial}>Imperial</Text>
  //         </TouchableOpacity>
  //       </View>
  //       <TouchableOpacity 
  //         onPress={() => registerFunc3()}
  //         style={styles.rect2}>
  //         <Text style={styles.next}>Next</Text>
  //       </TouchableOpacity>
  //     </ImageBackground>
  //   </View>
  // );
}

export default Registration03;
