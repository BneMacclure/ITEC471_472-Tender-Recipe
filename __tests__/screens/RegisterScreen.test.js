import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Registration from '../../screens/RegisterScreen';
import Registration02 from '../../screens/Registration02';
import Registration03 from '../../screens/Registration03';
import Registration04 from '../../screens/Registration04';
import Registration05 from '../../screens/Registration05';
import { Alert } from 'react-native';
import Enzyme, { configure, shallow, mount, render } from 'enzyme';
import {act, fireEvent, waitFor, cleanup, render} from '@testing-library/react-native';

describe('<RegisterScreen />', () => {
    it('RegisterScreen snapshot test' , () => {
        const tree = renderer.create(<Registration />).toJSON();
        expect(tree).toMatchSnapshot();
    });
	it('first registertion page renders',()=>{
		const nav=jest.fn();
		const param=jest.fn();
		const { getByText } = render(<Registration navigation={{nav}} route={{param}} />);
		fireEvent.press(getByText('Sign Up'));
		//expect(nav).toHaveBeenCalledWith('Registration02');
	});
	it('first registertion page renders',()=>{
		const wrapper = shallow(<Button />);
		const firstButton = wrapper.find(‘button’).at(0);
		const secondButton = wrapper.find(‘button’).at(1);
		
	});
	
	
	
	
	
})