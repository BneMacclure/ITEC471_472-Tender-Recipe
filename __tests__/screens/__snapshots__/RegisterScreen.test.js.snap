// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterScreen /> RegisterScreen snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "alignItems": "center",
        "height": 1334,
        "width": 750,
      }
    }
  >
    <Image
      resizeMode="cover"
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": 1334,
            "width": 750,
          },
          Object {},
        ]
      }
    />
    <Text
      style={
        Object {
          "color": "rgba(255,255,255,1)",
          "fontSize": 40,
          "marginTop": 44,
        }
      }
    >
      Registration
    </Text>
    <View
      style={
        Object {
          "marginTop": 100,
        }
      }
    />
    <View
      style={
        Object {
          "justifyContent": "center",
          "marginTop": 10,
          "width": 250,
        }
      }
    >
      <View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 6,
              },
              Object {
                "backgroundColor": "#E35614",
                "borderColor": "white",
                "borderRadius": 4,
                "borderWidth": 1,
              },
            ]
          }
        />
        <View
          style={
            Array [
              Object {
                "paddingBottom": 0,
              },
              Object {
                "minHeight": 64,
                "paddingTop": 8,
              },
            ]
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "bottom": 0,
                  "height": 2,
                  "left": 10,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 6,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                  "width": 8,
                }
              }
            />
            <Text
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "color": "transparent",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 18,
                  "opacity": 0,
                  "paddingHorizontal": 0,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 37,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                    Object {
                      "scaleY": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Name
            </Text>
            <Text
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Name
            </Text>
            <Text
              numberOfLines={1}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Name
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            color="#FFFFFF"
            editable={true}
            multiline={false}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholder=""
            placeholderTextColor="white"
            rejectResponderTermination={true}
            selectionColor="white"
            style={
              Array [
                Object {
                  "flexGrow": 1,
                  "margin": 0,
                  "paddingHorizontal": 14,
                  "zIndex": 1,
                },
                Object {
                  "height": 56,
                },
                Object {
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "textAlign": "left",
                  "textAlignVertical": "center",
                },
                Array [
                  Object {},
                ],
              ]
            }
            testID="namefield"
            underlineColorAndroid="transparent"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "justifyContent": "center",
          "marginTop": 10,
          "width": 250,
        }
      }
    >
      <View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 6,
              },
              Object {
                "backgroundColor": "#E35614",
                "borderColor": "white",
                "borderRadius": 4,
                "borderWidth": 1,
              },
            ]
          }
        />
        <View
          style={
            Array [
              Object {
                "paddingBottom": 0,
              },
              Object {
                "minHeight": 64,
                "paddingTop": 8,
              },
            ]
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "bottom": 0,
                  "height": 2,
                  "left": 10,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 6,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                  "width": 8,
                }
              }
            />
            <Text
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "color": "transparent",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 18,
                  "opacity": 0,
                  "paddingHorizontal": 0,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 37,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                    Object {
                      "scaleY": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Phone Number
            </Text>
            <Text
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Phone Number
            </Text>
            <Text
              numberOfLines={1}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Phone Number
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            clearTextOnFocus={true}
            editable={true}
            multiline={false}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholder=""
            placeholderTextColor="white"
            rejectResponderTermination={true}
            selectionColor="white"
            style={
              Array [
                Object {
                  "flexGrow": 1,
                  "margin": 0,
                  "paddingHorizontal": 14,
                  "zIndex": 1,
                },
                Object {
                  "height": 56,
                },
                Object {
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "textAlign": "left",
                  "textAlignVertical": "center",
                },
                Array [
                  Object {},
                ],
              ]
            }
            testID="phonefield"
            underlineColorAndroid="transparent"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "justifyContent": "center",
          "marginTop": 10,
          "width": 250,
        }
      }
    >
      <View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 6,
              },
              Object {
                "backgroundColor": "#E35614",
                "borderColor": "white",
                "borderRadius": 4,
                "borderWidth": 1,
              },
            ]
          }
        />
        <View
          style={
            Array [
              Object {
                "paddingBottom": 0,
              },
              Object {
                "minHeight": 64,
                "paddingTop": 8,
              },
            ]
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "bottom": 0,
                  "height": 2,
                  "left": 10,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 6,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                  "width": 8,
                }
              }
            />
            <Text
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "color": "transparent",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 18,
                  "opacity": 0,
                  "paddingHorizontal": 0,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 37,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                    Object {
                      "scaleY": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Email
            </Text>
            <Text
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Email
            </Text>
            <Text
              numberOfLines={1}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Email
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            autoCapitalize="none"
            editable={true}
            multiline={false}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholder=""
            placeholderTextColor="white"
            rejectResponderTermination={true}
            selectionColor="white"
            style={
              Array [
                Object {
                  "flexGrow": 1,
                  "margin": 0,
                  "paddingHorizontal": 14,
                  "zIndex": 1,
                },
                Object {
                  "height": 56,
                },
                Object {
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "textAlign": "left",
                  "textAlignVertical": "center",
                },
                Array [
                  Object {},
                ],
              ]
            }
            testID="emailfield"
            underlineColorAndroid="transparent"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "justifyContent": "center",
          "marginTop": 10,
          "width": 250,
        }
      }
    >
      <View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 6,
              },
              Object {
                "backgroundColor": "#E35614",
                "borderColor": "white",
                "borderRadius": 4,
                "borderWidth": 1,
              },
            ]
          }
        />
        <View
          style={
            Array [
              Object {
                "paddingBottom": 0,
              },
              Object {
                "minHeight": 64,
                "paddingTop": 8,
              },
            ]
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "bottom": 0,
                  "height": 2,
                  "left": 10,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 6,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                  "width": 8,
                }
              }
            />
            <Text
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "color": "transparent",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 18,
                  "opacity": 0,
                  "paddingHorizontal": 0,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 37,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                    Object {
                      "scaleY": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Password
            </Text>
            <Text
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Password
            </Text>
            <Text
              numberOfLines={1}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Password
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            autoCapitalize="none"
            editable={true}
            multiline={false}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholder=""
            placeholderTextColor="white"
            rejectResponderTermination={true}
            secureTextEntry={true}
            selectionColor="white"
            style={
              Array [
                Object {
                  "flexGrow": 1,
                  "margin": 0,
                  "paddingHorizontal": 14,
                  "zIndex": 1,
                },
                Object {
                  "height": 56,
                },
                Object {
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "textAlign": "left",
                  "textAlignVertical": "center",
                },
                Array [
                  Object {},
                ],
              ]
            }
            testID="passfield"
            underlineColorAndroid="transparent"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "justifyContent": "center",
          "marginTop": 10,
          "width": 250,
        }
      }
    >
      <View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 6,
              },
              Object {
                "backgroundColor": "#E35614",
                "borderColor": "white",
                "borderRadius": 4,
                "borderWidth": 1,
              },
            ]
          }
        />
        <View
          style={
            Array [
              Object {
                "paddingBottom": 0,
              },
              Object {
                "minHeight": 64,
                "paddingTop": 8,
              },
            ]
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "bottom": 0,
                  "height": 2,
                  "left": 10,
                  "opacity": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 6,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                  "width": 8,
                }
              }
            />
            <Text
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "#E35614",
                  "color": "transparent",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 18,
                  "opacity": 0,
                  "paddingHorizontal": 0,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 37,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                    Object {
                      "scaleY": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Confirm Password
            </Text>
            <Text
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Confirm Password
            </Text>
            <Text
              numberOfLines={1}
              style={
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "left": 0,
                  "opacity": 0,
                  "paddingHorizontal": 14,
                  "position": "absolute",
                  "textAlign": "left",
                  "top": 36,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateY": 0,
                    },
                    Object {
                      "scale": 1,
                    },
                  ],
                  "writingDirection": "ltr",
                }
              }
            >
              Confirm Password
            </Text>
          </View>
          <TextInput
            allowFontScaling={true}
            autoCapitalize="none"
            editable={true}
            multiline={false}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            placeholder=""
            placeholderTextColor="white"
            rejectResponderTermination={true}
            secureTextEntry={true}
            selectionColor="white"
            style={
              Array [
                Object {
                  "flexGrow": 1,
                  "margin": 0,
                  "paddingHorizontal": 14,
                  "zIndex": 1,
                },
                Object {
                  "height": 56,
                },
                Object {
                  "paddingBottom": 0,
                  "paddingTop": 0,
                },
                Object {
                  "color": "white",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": undefined,
                  "textAlign": "left",
                  "textAlignVertical": "center",
                },
                Array [
                  Object {},
                ],
              ]
            }
            testID="confirmpass"
            underlineColorAndroid="transparent"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "marginTop": 40,
        }
      }
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(255,255,255,1)",
          "borderColor": "#000000",
          "borderRadius": 25,
          "borderWidth": 0,
          "height": 59,
          "justifyContent": "center",
          "marginTop": 30,
          "opacity": 1,
          "width": 214,
        }
      }
      testID="signupbtn"
    >
      <Text
        style={
          Object {
            "color": "rgba(249,71,35,1)",
            "fontSize": 24,
          }
        }
      >
        Sign Up
      </Text>
    </View>
  </View>
</View>
`;
